### PR TITLE
Profile reform

### LINKED ISSUE #2: Code Reformat: Profile Page
DB call must be in useEffect

### PR DESCRIPTION
**Pull request by rpasricha45**
1. Fixed query / to remove method call 
2. added query to another query.js file  , and used variables for email field

### CODE REVIEW

**Review by brandonRodriguez24 — APPROVED** [review]
This looks good, query is just about the same as mine

**Review by vkoli — DISMISSED** [review]
Tested and works fine.
Can be merged to main

File: src/components/Profile.js [hunk 1]
```diff
@@ -20,27 +20,13 @@ const useStyles = makeStyles((theme) => ({
 const Profile = () => {
 	const classes = useStyles()
 	const { currentUser } = useAuth()
-	const [currentUserName, setName] = useState('')
 	const email = currentUser.email
+	const {loading ,error,data} = useQuery(GET_PROFILE_NAME,{
+		variables:{email}
+	})
```
Comment (vkoli) [inline src/components/Profile.js]: You want to pull the api call outside of the component.
The data that needs to rendered is dependent on a successful response

**Review by vkoli — CHANGES_REQUESTED** [review]

**Comment by vkoli** [comment]
After the discussion we had this morning, it would be better to try and limit the inconsistencies that may come up if the api request/response takes time.

**Comment by rpasricha45** [comment]
@vkoli  I added the api call outside of function to add more consistency to the code base as recommended

**Review by vkoli — APPROVED** [review]
Tested and works, nice job!

**Pull request merged**